systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -218,6 +219,7 @@ public class Environment {
     public static final String IP_FAMILY_DUAL_STACK = "dual";
 
     public static final String KAFKA_TIERED_STORAGE_BASE_IMAGE_DEFAULT = STRIMZI_REGISTRY_DEFAULT + "/" + STRIMZI_ORG_DEFAULT + "/kafka:latest-kafka-" + ST_KAFKA_VERSION_DEFAULT;
+    public static final String KANIKO_IMAGE_DEFAULT = "gcr.io/kaniko-project/executor:latest";
```

Comment:
For `KafkaConnect` build feature we use `gcr.io/kaniko-project/executor:v1.23.2`, wouldn't be better to use the same version as default? @im-konge  wdyt?

Reply:
Yeah I would stick with some specific version of Kaniko. There can be issues in the latest image, with specific version, we can do upgrade in some PR and fix there possible issues.

Reply:
okay, changed it. Thx